main/isotp_links.c

```diff
@@ -40,7 +40,8 @@ void configure_isotp_links()
     configure_isotp_link(2, 0x7E5, 0x7ED, "ptcu");
     configure_isotp_link(3, 0x607, 0x587, "gateway");
     configure_isotp_link(4, 0x749, 0x729, "awd");
-    configure_isotp_link(5, 0x744, 0x724, "suspension");
+    configure_isotp_link(5, 0x744, 0x724, "suspension1");
```

Comment:
i really need to make these configurable over the BLE layer like typical J2534 pass filters